installed_contracts/tokens/contracts/StandardToken.sol

```diff
@@ -39,22 +34,24 @@ contract StandardToken is Token {
     function pause()
         public
         onlyCreator
-        onlyLive
         returns (bool success)
     {
-        paused = true;
-        Paused();
+        if (paused == false) {
```

Comment:
I think it would be better to have this function fail if the contract is already paused rather than return success here even when changes don't take place.  Lets chat about why you made this change, would love to get your perspective.

Reply:
Yeah, I went back and forth on this as you can see. Ultimately I figured an owner trying to pause an already paused contract is a simple mistake, and shouldn't have the same nasty result (an error) as a non-owner trying to pause a contract.
I think the api is simpler this way as the owner isn't forced to worry about the state of `paused`. They can certainly check the state if they are curious, and it would be odd for an owner to be unsure of the state before trying to pause or resume, but I think it's nice that they aren't required to use it in their logic.
On the other hand, using `onlyLive` and `onlyPaused` modifiers for `pause()` and `resume()` and throwing an error when these cases are violated does have some benefits. It would help with debugging and testing. A function that returns success even when no changes were made and there was no reason to call it could cause frustration for a developer who is using this tool and cannot get their code to work properly.
I considered not using a modifier in this case but simply returning false if no changes are made, but this doesn't seem any better. That would require the developer to check the return value of the function, which leaves room for that same frustration.
After evaluating these three options, I agree with you that `pause()` and `resume()` should fail if they are already in that state, and I'll do this using modifiers.